calculator/calculator.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   
+    <link rel="stylesheet" href="css/styles.css">
+
+</head>
+
+<body>
```

Comment:
@paulghaddad I'm not sure why my "extra" file is showing? I added it to .gitignore....thoughts?